packages/flutter/lib/src/painting/image_resolution.dart

```diff
@@ -14,6 +13,7 @@ import 'package:flutter/services.dart';
 import 'image_provider.dart';
 
 const String _kAssetManifestFileName = 'AssetManifest.json';
+const String _kSeparator = '/';
```

Comment:
cc @goderbauer do you know if we always use file uris for assets? Or do windows paths end up with a `\` ?

Reply:
I don't know :(

Reply:
I don't think they are file paths, if it were the case assets wouldn't work if you built on windows for an Android device